internal/mode/static/nginx/config/http/config.go

```diff
@@ -12,12 +12,15 @@ type Server struct {
 
 // Location holds all configuration for an HTTP location.
 type Location struct {
-	Return          *Return
-	Rewrites        []string
-	Path            string
-	ProxyPass       string
-	HTTPMatchVar    string
-	ProxySetHeaders []Header
+	Return                *Return
+	Rewrites              []string
+	Path                  string
+	ProxyPass             string
+	HTTPMatchVar          string
+	ProxySetHeaders       []Header
+	AddResponseHeaders    []Header
```

Comment:
We might consider reusing `dataplane.HTTPHeaderFilter` instead of using `AddResponseHeaders`, `SetResponseHeaders`, and `RemoveResponseHeaders`. Why do we need both `Header` and `dataplane.HTTPHeader`?